src/applications/vaos/services/appointment/transformers.js

```diff
@@ -355,25 +355,28 @@ function setParticipant(appt) {
 
   switch (type) {
     case APPOINTMENT_TYPES.vaAppointment: {
-      const participant = [];
+      let participant = {};
       if (appt.clinicId) {
-        participant.push({
-          actor: {
+        participant = {
```

Comment:
I think we could do more to simplify the participant object here. There are only a few fields we need, we could do something like:

```
{
   vistaId: appt.facilityId,
   clinicId: appt.clinicId,
   stationId: appt.sta6aid,
   clinicName: appt.clinicFriendlyName ||
     appt.vdsAppointments?.[0]?.clinic?.name ||
     appt.vvsAppointments?.[0]?.clinic?.name,
}
```

"Participant" is a FHIR concept that doesn't necessarily mean anything to us, too. So we don't necessarily have to have these fields in a separate `participant` object either, though if we want to keep that for now, it's ok with me.

Reply:
Done.